resolwe_bio/tools/bed_file_corrections_genome_browsers.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+"""Restrict all scores to the maximal value of 1000."""
+
+import argparse
+import pandas as pd
+
+parser = argparse.ArgumentParser(description="Restrict all scores to the maximal value of 1000.")
+parser.add_argument('-f', '--bed_file', help="Bed file.")
+args = parser.parse_args()
+
+df = pd.read_csv(args.bed_file, delimiter='\t', header=None, dtype=str)
+df.iloc[:, 4] = pd.to_numeric(df[df.columns[4]]).round().astype(int)
```

Comment:
`df[df.columns[4]]` on the right side of ` = ` can be improved as well.

Reply:
how?

Reply:
Change to `df.iloc[:, 4]` https://github.com/genialis/resolwe-bio/pull/576#discussion_r194338150.